FraihmworkHealthAndIntegrityMonitoringApi.yaml

```diff
@@ -173,6 +173,50 @@ paths:
       security:
       - SystemWrite:
         - fraihmwork.system.write
+   
+  /system/{uuid}/refresh:
+    post:
+      tags:
+      - System
+      summary: Refresh existing system properties
+      description: Permits an already registered system to be refreshed without changing any properties
+      operationId: refreshSystemInformation
+      parameters:
+        - in: path
+          name: uuid
+          required: true
+          schema:
+            type: string
+            format: uuid
+          description: Uuid of the system property being refreshed
+      responses:
+        200:
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'

```

Comment:
Should /Response be lowercase?

Reply:
According to https://swagger.io/specification/, elements under /components/schemas are uppercase.

Reply:
No, it's uppercase itself.